WordPressKit/ActivityServiceRemote.swift

```diff
@@ -67,6 +67,38 @@ open class ActivityServiceRemote: ServiceRemoteWordPressComREST {
                                     failure(error)
                                 })
     }
+    
+    /// Retrieves activity groups associated with a site.
+    ///
+    /// - Parameters:
+    ///     - siteID: The target site's ID.
+    ///     - success: Closure to be executed on success.
+    ///     - failure: Closure to be executed on error.
+    ///
+    /// - Returns: An array of available activity groups for a site.
+    ///
+    open func getActivityGroupsForSite(_ siteID: Int,
```

Comment:
We also need two arguments here: `before` and `after`, just like the getActivities. When the user filters by a date range we pass this into this request so only groups inside the given range are returned.

Another thing is that if we pass just a `after` OR `before` we should make the request using `on`. Eg.:

`https://public-api.wordpress.com/wpcom/v2/sites/165243437/activity/count/group?_envelope=1&on=2020-11-17%2003%3A00%3A00&number=1000`

Reply:
Fixed! 6a7698b